system/micd.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+import time
+
+import sounddevice as sd
+import numpy as np
+
+from cereal import messaging
+from common.filter_simple import FirstOrderFilter
+from common.realtime import Ratekeeper
+from system.swaglog import cloudlog
+
+RATE = 10
+DT_MIC = 1. / RATE
+
+MUTE_TIME = 5
+
+
+class Mic:
+  def __init__(self, pm, sm):
+    self.pm = pm
+    self.sm = sm
+    self.rk = Ratekeeper(RATE)
+
+    self.measurements = np.array([])
+    self.filter = FirstOrderFilter(1, 7, DT_MIC)
+    self.last_alert_time = 0
+
+  def update(self):
+    self.sm.update(0)
+
+    if self.sm.updated['controlsState']:
+      if self.sm['controlsState'].alertSound > 0:
+        self.last_alert_time = time.time()
```

Comment:
hmm, things in system/ shouldn't subscribe to anything in selfdrive/. can we do this more generically? if not, perhaps move to selfdrive/ for now